frontend/src/components/announcements/AnnouncementBlock.js

```diff
@@ -14,23 +16,27 @@ class AnouncementBlock extends React.Component {
     componentDidMount(){
         fetch(url)
         .then(response => response.json())
+        .then(data => data.reverse())
         .then(data => this.setState({ announcementList : data}));
-        
     }
 
     render(){
         return(
-            <div id="announce">
+            <div>
                 {this.state.announcementList && this.state.announcementList.map(a => {
                     const title = a.title;
                     const author = a.author;
                     const content = a.content;
-    
+                    const date = dateFormat(a.date, "mmmm dS, hh:mm");
```

Comment:
When testing, I noticed that the date here is off. It is only posting in on the fourth minute of every hour (7:04, 8:04). One other suggestion that I have is to include AM or PM in this time stamp.